src/code42cli/cmds/users.py

```diff
@@ -60,6 +61,50 @@ def list_users(state, org_uid, role_name, active, inactive, format):
         formatter.echo_formatted_dataframe(df)
 
 
+@users.command()
+@click.argument("username")
+@click.argument("role-name")
+@sdk_options()
+def add_role(state, username, role_name):
+    """Add the specified role to the specified username"""
+    _add_user_role(state.sdk, role_name, username)
+
+
+@users.command()
+@click.argument("username")
+@click.argument("role-name")
+@sdk_options()
+def remove_role(state, username, role_name):
+    """Add the specified role to the specified username"""
+    _remove_user_role(state.sdk, role_name, username)
+
+
+def _add_user_role(sdk, role_name, username):
+    user_id = _get_user_id(sdk, username)
+    try:
+        _get_role_id(sdk, role_name)
+    except Code42CLIError:
+        raise
+    sdk.users.add_role(user_id, role_name)
+
+
+def _remove_user_role(sdk, role_name, username):
+    user_id = _get_user_id(sdk, username)
+    try:
+        _get_role_id(sdk, role_name)
+    except Code42CLIError:
+        raise
+    sdk.users.remove_role(user_id, role_name)
+
+
+def _get_user_id(sdk, username):
+    user = sdk.users.get_by_username(username)["users"]
+    if not user:
+        raise UserDoesNotExistError
```

Comment:
We have a share method that handles this nicely, located in `code42cli.cmds.shared.get_user_id()`... It will even cache IDs for users! We should use that instead.

Also, when raising errors, you need to use an instance of the error, like `UserDoesNotExistError(username)` (see `code42cli.cmds.shared.get_user_id()`)

Reply:
Ah I just noticed that this uses the legacy ID so that won't work.

However, we still need to raise an instance of the error here